### PR TITLE
Update Ui to display field name for clarity

### LINKED ISSUE #96: Add method of distinguishing fields
General comment: Not for this PR but we should probably eventually use colours or just indicate field names for these alternative fields, since with this PR it's no longer clear what fields means what. E.g. while I could infer that `@asdasd` is Bernice's telegram handle, at a glance I can't tell what `asd` represents for Alex

image

_Originally posted by @kokrui in https://github.com/AY2324S1-CS2103T-T13-2/tp/issues/95#issuecomment-1778804953_

### PR DESCRIPTION
**Pull request by limjunxian1**
This PR will update the Ui as shown below
image
Closes #96

### CODE REVIEW

File: src/main/java/seedu/address/ui/PersonCard.java
Comment (kokrui) [inline src/main/java/seedu/address/ui/PersonCard.java:91]: This wasn't detected as a merge conflict by GitHub, but it definitely is one. This modification would roll back the changes in https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/110, which we don't want. Maybe try merging master to this branch!

**Review by kokrui — CHANGES_REQUESTED** [review]
Great changes -- the new feature / ui change looks good to me overall! However, there's an undetected-by-GitHub and thus unresolved merge conflict which must be fixed before this can be merged. Happy to approve once that's fixed!

File: src/main/java/seedu/address/ui/PersonCard.java
Comment (limjunxian1) [reply src/main/java/seedu/address/ui/PersonCard.java:91]: thanks for pointing this out! apparently when I merged into the master branch again, seems like it doesn't detect merge conflict so I changed it manually! Fixed in d6c95a1

**Review by kokrui — APPROVED** [review]
Thanks for the edits, LGTM! Though I think there were some markbind log files accidentally committed that should be removed before u merge

**Comment by limjunxian1** [comment]
yes, thank you for pointing that out! deleted in 0aa38ed

**Pull request merged**
